config/config.go

```diff
@@ -27,6 +27,7 @@ type Settings struct {
 	ClientSecret         string `envconfig:"client_secret" required:"true"`
 	DefaultOrigin        string `envconfig:"default_origin" required:"true"`
 	Schedule             string `envconfig:"schedule" default:"0 0 * * * *"`
+	UserIdPool					 []string `envconfig:"user_id_pool" required:"true"` 
```

Comment:
The formatting is off and could be fixed. Run `gofmt -w` on the file and it'll autofix it.

Reply:
fixed